frontend/src/js-admin/utils.ts

```diff
@@ -0,0 +1,4 @@
+export function ISOToDateString(ISOString: string) {
+  var dateObj = new Date(ISOString);
+  return dateObj.getDate() + '.' + dateObj.getMonth();
```

Comment:
oops, didn't notice before: this is incorrect, `getMonth()` is *zero-indexed*, i.e. January = 0, Feb = 1 and so on.

I'd recommend using `moment` to handle the formatting, we're already using it in the project, like this: https://github.com/teekkarispeksi/nappikauppa2/blob/master/backend/src/show.ts#L66